### DISCUSSION TITLE
Make this discussion forums easier to be found

### DISCUSSION THREAD

**Discussion by taoning2014** [npm/feedback#638] [Website] [answered]
Hi Friends 👋 ! 

I'm trying to give some suggestions on the NPM CLI, however, it takes me certain efforts to find this amazing forum. 

I noticed there is a [support page](https://www.npmjs.com/support) on the npm website which is where I first headed to. But inside that page, it linked to the generic Github dev community forum.

Maybe you can replace that link with this more relevant discussion form?

<img width="1904" alt="npm-support" src="https://user-images.githubusercontent.com/8691648/153709325-d1fec488-d745-4a31-b411-b0aa4f5b61b4.png">

**Comment by taoning2014** [comment]
Another place that I noticed it can be updated is https://npm.community:

<img width="1904" alt="npm-community" src="https://user-images.githubusercontent.com/8691648/153709703-47d30b42-301f-40af-b532-648af5b653da.png">

**Comment by MylesBorins** [comment, marked as answer]
Thanks for the suggestions! I have an open PR to add documentation about our feedback process. We are not going to change the links you suggested as we still want customers to use the GitHub Community Form for general support + help, this feedback process is only for proposing new ideas and changes to npm.
